hs_file_types/utils.py

```diff
@@ -307,54 +332,119 @@ def ingest_logical_file_metadata(metadata_file, resource, map_files):
     if not agg_type_name:
         raise Exception("Could not derive aggregation type from {}".format(metadata_file.name))
     subject = None
-    for s, _, _ in graph.triples((None, DC.title, None)):
+    for s, _, _ in graph.triples((None, DC.type, None)):
         subject = s.split('/resource/', 1)[1].split("#")[0]
         break
     if not subject:
         raise Exception("Could not derive aggregation path from {}".format(metadata_file.name))
 
     logical_file_class = get_logical_file(agg_type_name)
     lf = get_logical_file_by_map_file_path(resource, logical_file_class, subject)
-
+    logical_files = {}
```

Comment:
I'm trying to resolve the merge conflicts and get your changes to the metadata ingestion working.  Some of these changes read like a resource file can be a part of multiple aggregations... that does not seem right.